docs/clusterfuzzlite/build_integration.md

```diff
@@ -0,0 +1,286 @@
+---
```

Comment:
Hard to say if we should go that way without actually seeing it. Worth a try, but we don't want to get into a state where it's difficult to predict what the output will be from the input files that we edit.

Reply:
There are quite a few, I'm not sure we can reasonably do this:
1. Every command is different because ClusterFuzzLite uses $PATH_TO_PROJECT and --external. OSS-Fuzz doesn't
2. Examples are different.
3. ClusterFuzzLite does not mention project.yaml
4. ClusterFuzzLite does not mention MSAN.
5. Example docker code is different.
6. OSS-Fuzz string is replaced with ClusterFuzzLite.
7. ClusterFuzzLite mentions a few things are the same in OSS-Fuzz for this familiar (e.g. build.sh).
8. ClusterFuzzLite doesn't mention diskspace because it is different depending on CI.
9. ClusterFuzzLite does not mention checking into OSS-Fuzz repo.
10. ClusterFuzzLite does not mention license/copyright headers.
11. ClusterFuzzLite does not mention status badge (yet).
12. ClusterFuzzLite does not mention OSS-Fuzz website